modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectValidationRuleLocalServiceTest.java

```diff
@@ -119,77 +219,154 @@ public void testDeleteObjectValidationRule() throws Exception {
-
-		try {
-			objectValidationRule =
-				_objectValidationRuleLocalService.updateObjectValidationRule(
-					RandomTestUtil.randomLong(), false,
-					ObjectValidationRuleConstants.ENGINE_TYPE_DDM,
-					LocalizedMapUtil.getLocalizedMap(
-						RandomTestUtil.randomString()),
-					LocalizedMapUtil.getLocalizedMap(
-						RandomTestUtil.randomString()),
-					ObjectValidationRuleConstants.OUTPUT_TYPE_FULL_VALIDATION,
-					_VALID_DDM_SCRIPT, Collections.emptyList());
-
-			Assert.fail();
-		}
-		catch (NoSuchObjectValidationRuleException
-					noSuchObjectValidationRuleException) {
-
-			Assert.assertNotNull(noSuchObjectValidationRuleException);
-		}
+		ObjectValidationRule objectValidationRule1 = _addObjectValidationRule(
+			ObjectValidationRuleConstants.ENGINE_TYPE_DDM,
+			ObjectValidationRuleConstants.OUTPUT_TYPE_FULL_VALIDATION,
+			Collections.emptyList(), _VALID_DDM_SCRIPT);
+
+		long randomId = RandomTestUtil.randomLong();
+
+		AssertUtils.assertFailure(
+			NoSuchObjectValidationRuleException.class,
+			String.format(
+				"No ObjectValidationRule exists with the primary key %s",
+				randomId),
+			() -> _objectValidationRuleLocalService.updateObjectValidationRule(
+				randomId, false, ObjectValidationRuleConstants.ENGINE_TYPE_DDM,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				ObjectValidationRuleConstants.OUTPUT_TYPE_FULL_VALIDATION,
+				_VALID_DDM_SCRIPT, Collections.emptyList()));
 
-		objectValidationRule =
+		objectValidationRule1 =
 			_objectValidationRuleLocalService.updateObjectValidationRule(
-				objectValidationRule.getObjectValidationRuleId(), true,
+				objectValidationRule1.getObjectValidationRuleId(), true,
 				ObjectValidationRuleConstants.ENGINE_TYPE_DDM,
 				LocalizedMapUtil.getLocalizedMap("Field must be an URL"),
 				LocalizedMapUtil.getLocalizedMap("URL Validation"),
 				ObjectValidationRuleConstants.OUTPUT_TYPE_FULL_VALIDATION,
 				"isURL(textField)", Collections.emptyList());
 
-		Assert.assertTrue(objectValidationRule.isActive());
+		Assert.assertTrue(objectValidationRule1.isActive());
 		Assert.assertEquals(
 			ObjectValidationRuleConstants.ENGINE_TYPE_DDM,
-			objectValidationRule.getEngine());
+			objectValidationRule1.getEngine());
 		Assert.assertEquals(
 			LocalizedMapUtil.getLocalizedMap("Field must be an URL"),
-			objectValidationRule.getErrorLabelMap());
+			objectValidationRule1.getErrorLabelMap());
 		Assert.assertEquals(
 			LocalizedMapUtil.getLocalizedMap("URL Validation"),
-			objectValidationRule.getNameMap());
+			objectValidationRule1.getNameMap());
 		Assert.assertEquals(
-			"isURL(textField)", objectValidationRule.getScript());
+			"isURL(textField)", objectValidationRule1.getScript());
+
+		List<ObjectField> objectFields =
+			_objectFieldLocalService.getObjectFields(
+				_objectDefinition.getObjectDefinitionId(), false);
+
+		ObjectField objectField = objectFields.get(0);
```

Comment:
I was thinking, we can't use this get:

```java
_objectFieldLocalService.fetchObjectField(
			_objectDefinition.getObjectDefinitionId(), {name of field});
```
To get the specific field that we want?

Reply:
As discussed, we'll only do this for the first one